src/rt/cover.d

```diff
@@ -149,10 +149,84 @@ uint parseNum(const(char)[] s)
 T min(T)(T a, T b) { return a < b ? a : b; }
 T max(T)(T a, T b) { return b < a ? a : b; }
 
+version (unittest) __gshared bool inUnittest;
+
+// Parse a string looking like a boolean and assign the result to the global
+// merge variable.
+string parseMergeOpt(string opt) @nogc nothrow
+{
+    switch (opt)
+    {
+        case "no", "No", "NO", "n", "N", "0":
+            merge = false;
+            return "0";
+        case "", "yes", "Yes", "YES", "y", "Y", "1":
+            merge = true;
+            return "1";
+        default:
+            version (unittest) if (inUnittest) return "err";
+            fprintf(stderr, "DRT Error: Runtime option 'covmerge' " ~
+                    "should be 1/y/yes or 0/n/no, got '%.*s'.\n",
+                    cast(int)opt.length, opt.ptr);
+            return "err";
+    }
+}
+
+unittest
+{
+    inUnittest = true;
+    scope (exit) inUnittest = false;
+
+    bool old_merge = merge;
+    scope (exit) merge = old_merge;
+
+    merge = true;
+    assert(parseMergeOpt("0") == "0" && !merge);
+    assert(parseMergeOpt("1") == "1" && merge);
+    assert(parseMergeOpt("n") == "0" && !merge);
+    assert(parseMergeOpt("y") == "1" && merge);
+    assert(parseMergeOpt("N") == "0" && !merge);
+    assert(parseMergeOpt("Y") == "1" && merge);
+    assert(parseMergeOpt("No") == "0" && !merge);
+    assert(parseMergeOpt("Yes") == "1" && merge);
+    assert(parseMergeOpt("NO") == "0" && !merge);
+    assert(parseMergeOpt("YES") == "1" && merge);
+    assert(parseMergeOpt("no") == "0" && !merge);
+    assert(parseMergeOpt("yes") == "1" && merge);
+    assert(parseMergeOpt("no") == "0" && !merge);
+    assert(parseMergeOpt("") == "1" && merge);
+    merge = false;
+    assert(parseMergeOpt("yEs") == "err" && !merge);
+    merge = true;
+    assert(parseMergeOpt("nO") == "err" && merge);
+    merge = false;
+    assert(parseMergeOpt("10") == "err" && !merge);
+    merge = true;
+    assert(parseMergeOpt("01") == "err" && merge);
+    assert(parseMergeOpt("xxx") == "err" && merge);
+    assert(parseMergeOpt("invalid") == "err" && merge);
+}
+
+// Parse options taken from the standard runtime options passing via
+// --DRT-xxx / DRT_XXX.
+void updateDRTOpts()
+{
+    import core.internal.traits : externDFunc;
+    alias rt_configCallBack = string delegate(string) @nogc nothrow;
+    alias fn_configOption = string function(string opt, scope rt_configCallBack dg, bool reverse) @nogc nothrow;
+    alias rt_configOption = externDFunc!("rt.config.rt_configOption", fn_configOption);
```

Comment:
Why not simply `import rt.config : rt_configOption;`

Reply:
I don't know. I just copied what it's done in `gc/config.d` thinking that there was a good reason for it.